simtools/io_handler.py

```diff
@@ -32,10 +32,15 @@ def __init__(self):
         self._logger.debug("Init IOHandler")
 
         self.output_path = None
+        self.plain_output_path = False
         self.data_path = None
         self.model_path = None
 
-    def set_paths(self, output_path=None, data_path=None, model_path=None):
+    def set_paths(self,
+                  output_path=None,
+                  data_path=None,
+                  model_path=None,
+                  plain_output_path=False):
```

Comment:
Commenting here (out of place): Please add to the Parameters doc string the new parameter

Reply:
Done.